BitwardenShared/UI/Vault/Extensions/Alert+Vault.swift

```diff
@@ -183,119 +183,118 @@ extension Alert {
-                    await action(.copyTotp(
-                        totpKey: TOTPKeyModel(authenticatorKey: totp),
-                        requiresMasterPasswordReprompt: cipherView.reprompt == .password && hasMasterPassword
-                    ))
-                })
-            }
-            if let uri = cipherView.login?.uris?.first?.uri,
-               let url = URL(string: uri) {
-                alertActions
-                    .append(AlertAction(title: Localizations.launch, style: .default) { _, _ in
-                        await action(.launch(url: url))
-                    })
-            }
-        case .identity:
-            // No-op: no extra options beyond view and edit.
-            break
-        case .secureNote:
-            if let notes = cipherView.notes {
-                alertActions.append(AlertAction(title: Localizations.copyNotes, style: .default) { _, _ in
-                    await action(.copy(
-                        toast: Localizations.notes,
-                        value: notes,
-                        requiresMasterPasswordReprompt: false,
-                        logEvent: nil,
-                        cipherId: nil
-                    ))
-                })
-            }
-        case .sshKey:
-            if let sshKey = cipherView.sshKey {
-                alertActions.append(AlertAction(title: Localizations.copyPublicKey, style: .default) { _, _ in
-                    await action(.copy(
-                        toast: Localizations.publicKey,
-                        value: sshKey.publicKey,
-                        requiresMasterPasswordReprompt: false,
-                        logEvent: nil,
-                        cipherId: cipherView.id
-                    ))
-                })
-                if cipherView.viewPassword {
-                    alertActions.append(AlertAction(title: Localizations.copyPrivateKey, style: .default) { _, _ in
-                        await action(.copy(
-                            toast: Localizations.privateKey,
-                            value: sshKey.privateKey,
-                            requiresMasterPasswordReprompt: cipherView.reprompt == .password && hasMasterPassword,
-                            logEvent: nil,
-                            cipherId: cipherView.id
-                        ))
-                    })
-                }
-                alertActions.append(AlertAction(title: Localizations.copyFingerprint, style: .default) { _, _ in
-                    await action(.copy(
-                        toast: Localizations.fingerprint,
-                        value: sshKey.fingerprint,
-                        requiresMasterPasswordReprompt: false,
-                        logEvent: nil,
-                        cipherId: cipherView.id
-                    ))
-                })
-            }
-        }
+        //       // Add any additional actions for the type of cipher selected.
```

Comment:
TODO: We probably need to adjust this to be lazy vs eager, and ad-hoc decrypt items when clicking on items.